addon/services/keyboard.js

```diff
@@ -65,6 +69,15 @@ export default class KeyboardService extends Service {
 
   @action
   _respond(event) {
+    if (this._disableOnInput && event.target) {
```

Comment:
@st-h @lukemelia I think it may be more flexible to be able to configure this per modifier instance, however tat may be done as a next step enhancement